### ISSUE TITLE
Cannot install dplyr from github

### ISSUE THREAD

**Issue by hoxo-m** [hadley/dplyr#1083]
I did `install_github("hadley/dplyr")`, but it didn't succeed.
Then I tried `install_github("hadley/dplyr", ref = "ad19afee89b6c5881fba6f2d364dfd0a585e5a93")`
that was sucessful.

```
R version 3.1.2 (2014-10-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)

locale:
[1] LC_COLLATE=Japanese_Japan.932 
[2] LC_CTYPE=Japanese_Japan.932   
[3] LC_MONETARY=Japanese_Japan.932
[4] LC_NUMERIC=C                  
[5] LC_TIME=Japanese_Japan.932    

attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

loaded via a namespace (and not attached):
[1] devtools_1.7.0.9000 evaluate_0.5.5     
[3] formatR_1.0         httr_0.5           
[5] knitr_1.9           RCurl_1.95-4.3     
[7] stringr_0.6.2       tools_3.1.2  
```

Error messages:

```
* installing *source* package 'dplyr' ...
** libs

*** arch - i386
g++ -m32 -I"C:/PROGRA~1/R/R-31~1.2/include"       -I../inst/include -DCOMPILING_DPLYR   -I"C:/Users/***/Documents/R/win-library/3.1/Rcpp/include" -I"C:/Users/***/Documents/R/win-library/3.1/BH/include" -I"d:/RCompile/CRANpkg/extralibs64/local/include"     -O3 -Wall  -mtune=core2      -c RcppExports.cpp -o RcppExports.o
In file included from ../inst/include/dplyr/Result/all.h:21:0,
                 from ../inst/include/dplyr.h:147,
                 from RcppExports.cpp:4:
../inst/include/dplyr/Result/DelayedProcessor.h: In function 'dplyr::DelayedProcessor_Base<CLASS, Data>* dplyr::get_delayed_processor(SEXP, int)':
../inst/include/dplyr/Result/DelayedProcessor.h:140:59: error: 'x' was not declared in this scope
make: *** [RcppExports.o] Error 1
 警告:  命令 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-31~1.2/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-31~1.2/share/make/winshlib.mk" -f "C:/Users/***/Documents/.R/Makevars" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="dplyr.dll" OBJECTS="RcppExports.o address.o arrange.o between.o bind.o combine_variables.o distinct.o dplyr.o filter.o group_indices.o init.o join.o select.o strings_addresses.o summarise.o test.o window.o"' の実行は状態 2 を持ちました  
ERROR: compilation failed for package 'dplyr'
```

**Comment by hoxo-m** [comment]
Resolved by commit [434b9ed82b57397ee5ee9d8f8e2ff372e5d8043b](https://github.com/hadley/dplyr/commit/434b9ed82b57397ee5ee9d8f8e2ff372e5d8043b)

**Comment by bergsmat** [comment]
At the moment, I cannot install_github('dplyr') on Windows (8), but using ref= works with the commit cited above. Error for head of master is something about wrong number of template arguments.
